requestium/requestium.py

```diff
@@ -89,6 +90,18 @@ def _start_chrome_browser(self):
             else:
                 raise Exception('A list is needed to use \'arguments\' option. Found {}'.format(
                     type(self.webdriver_options['arguments'])))
+                
+        # adds in ability to set a new download directory
+        # https://stackoverflow.com/questions/35331854/downloading-a-file-at-a-specified-location-through-python-and-selenium-using-chr/43789674#43789674
+        if 'prefs' in self.webdriver_options:
+            if isinstance(self.webdriver_options['prefs'], dict):
```

Comment:
This can be `and`ed with the previous line, and save one indent level below.

Reply:
You could also directly do `isinstance(self.webdriver_options.get('prefs'), dict)` which will check for both conditions (since `.get` returns `None` if not present).